cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,26 @@
+class PaymentStepPage{
+
+    private bankWire: string;
+    private confirmPay: string;
+    private finishShopping: string;
+
+    constructor(){
+        this.bankWire = "[class=bankwire]";
+        this.confirmPay = "#cart_navigation > button";
+        this.finishShopping = "#center_column > div > p > strong";
+    }
+
+    public payingWire(): void {
```

Comment:
name of methods should use a verb in infinitive mode: For example, this method can be named as: payByBankWire or selectBankWirePayment or something similar